x-pack/filebeat/modules.d/oracle.yml.disabled

```diff
@@ -0,0 +1,13 @@
+# Module: oracle
+# Docs: https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-oracle.html
```

Comment:
This is causing CI failure:

```diff
diff --git a/x-pack/filebeat/modules.d/oracle.yml.disabled b/x-pack/filebeat/modules.d/oracle.yml.disabled
index 0a7819e55..3bd576ee1 100644
--- a/x-pack/filebeat/modules.d/oracle.yml.disabled
+++ b/x-pack/filebeat/modules.d/oracle.yml.disabled
@@ -1,5 +1,5 @@
 # Module: oracle
-# Docs: https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-oracle.html
+# Docs: https://www.elastic.co/guide/en/beats/filebeat/7.x/filebeat-module-oracle.html

 - module: oracle
   database_audit:
```

need to run `mage update` in `x-pack/filebeat`